File: problems/t/tarifa/tarifa.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>

using namespace std;

int main() {
    int x, n, p, internet;

    cin >> x;
    cin >> n;

    internet = x;

    for (int i=0; i < n; i++) {
        cin >> p;
        internet += (x - p);
    }

    cout << internet << endl;
}

File: problems/t/tarifa/tarifa.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    x = int(input())
    n = int(input())

    internet = x
    for _ in range(n):
        p = (int(input()))
        internet += (x - p)

    print(internet)

if __name__ == '__main__':
    main()